Form/Type/ColorSelectorType.php

```diff
@@ -48,7 +48,15 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::WHITE,
                 Colors::YELLOW,
             ),
-        ));
+        );
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $defaults['choices'] = array_unique(array_flip(Colors::getAll()));
```

Comment:
:smile: 
Maybe like this, if we think that color code should be the key of array (2.8+) invert the array should be done just for bellow Symfony versions.
```php
    /**
     * Return the list of colors.
     *
     * @return array
     */
    public static function getAll($isInverted = false)
    {
        $colors = array(
            self::ALICEBLUE => 'aliceblue',
            self::ANTIQUEWHITE => 'antiquewhite',
            //...
        );

        // NEXT_MAJOR: Remove this "if" (when requirement of Symfony is >= 2.8)
        if($isInverted) {
            @trigger_error(
                'The $isInverted argument of '.__METHOD__.' is deprecated since 3.3, to be removed in 4.0.',
                E_USER_DEPRECATED
            );
            return array_flip($colors);
        }
        
        return $colors;
    }
```
And we should remove the 2 duplicate ``const`` colors ``#FF00FF`` (MAGENTA/FUCHSIA) and ``#00FFFF`` (AQUA/CYAN)